README.md

```diff
@@ -10,23 +10,64 @@ We are using [backport project](https://backports.wiki.kernel.org/index.php/Main
  We will add a new branch suse/sles<x.y> whenever a version is deprecated or moved to the maintenance phase.
```

Comment:
Include "This repo is a code snapshot of particular version of backports and does not contain individual change git history."

Reply:
done